foundryup/install

```diff
@@ -57,6 +57,11 @@ if [[ "$FOUNDRYUP_REPO" == "gakonst/foundry" && -z "$FOUNDRYUP_BRANCH" ]]; then
     ARCHITECTURE="amd64" # Amd.
   fi
   
+  # Warn MacOS users that they may need to manually install libusb via Homebrew:
+  if [[ "$PLATFORM" == "darwin" && ! -f /usr/local/opt/libusb/lib/libusb-1.0.0.dylib ]]; then
+    echo "warning: libusb not found. You may need to install it manually on MacOS via Homebrew (`brew install libusb`).
+  fi
+  
```

Comment:
fyi, this warning doesn't pop up on foundry install, maybe because `echo $PLATFORM` returns nothing

Reply:
`$OSTYPE` could work though

```zsh
% echo $OSTYPE
darwin20.0
```

Reply:
oh just realizing you removed that warning entirely also. https://github.com/gakonst/foundry/pull/537

Reply:
feel free :smile: